MediaBrowser.Model/Dlna/SingleAudioMode.cs

```diff
@@ -0,0 +1,28 @@
+namespace MediaBrowser.Model.Dlna
+{
+    /// <summary>
+    /// Track selection mode when track switching is not supported.
+    /// </summary>
+    public enum SingleAudioMode
```

Comment:
Or `SingleAudioPolicy`? 🤔
Any suggestions?

Reply:
I think `SingleAudioPolicy` describes it better